addons/project/static/src/js/tours/project.js

```diff
@@ -162,25 +162,34 @@ registry.category("web_tour.tours").add('project_tour', {
     position: "bottom",
     run: "click",
 },
-
 {
     trigger: ".o_form_project_tasks",
     isActive: ["auto"],
 },
 {
+    isActive: ["auto"],
     trigger: ".o_field_widget[name='user_ids'] input",
     content: _t("Assign a responsible to your task"),
     position: "right",
-    run: "edit Mitch",
+    run: "edit Dummy && clear",
 },
 {
-    isActive: ["auto", "desktop"],
-    trigger: ".o_field_widget[name='user_ids'] .o-autocomplete--dropdown-menu li:contains(Mitchell Admin)",
+    isActive: ["manual"],
+    trigger: ".o_field_widget[name='user_ids'] input",
+    content: _t("Assign a responsible to your task"),
+    position: "right",
+    run: "click",
+},
+{
+    isActive: ["desktop"],
+    trigger: "a.dropdown-item[id*='user_ids'] span",
+    content: markup(_t("Select an <b>assignee</b> from the menu")),
+    position: "right",
     run: "click",
 },
 {
     isActive: ["mobile"],
-    trigger: "div[role='article']",
+    trigger: "div.o_kanban_renderer > article.o_kanban_record",
```

Comment:
This change in trigger is due to change in kanban view and due to this auto tour run in mobile mode would fail.